test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -364,7 +365,10 @@ describe("GET tests", () => {
     expect(resp.text).toContain(SERVICE_ADDRESS_SAME_AS_PRINCIPAL_ADDRESS_TEXT);
   });
 
-  test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with trust data`, async () => {
+  test(`renders the ${CHECK_YOUR_ANSWERS_PAGE} page with trust data and feature flag off`, async () => {
+
```

Comment:
should not be new line before `}` or after `{`

Reply:
Completed